parser.y

```diff
@@ -10771,4 +10786,95 @@ LoadStatsStmt:
 		}
 	}
 
+/********************************************************************************************
+ *
+ *  Create Sequence Statement
+ *
+ *  Example:
+ *      CREATE SEQUENCE IF NOT EXIST my_sequence
+ *          [INCREMENT BY <increment>]
+ *          [START WITH <start>]
+ *	    [MINVALUE <minvalue> | NOMINVALUE]
+ *          [MAXVALUE <maxvalue> | NOMAXVALUE]
+ *          [CACHE <cache>]
+ *          [CYCLE | NO CYCLE]
+ *          [ORDER | NO ORDER]
+ ********************************************************************************************/
+
+CreateSequenceStmt:
+	"CREATE" "SEQUENCE" IfNotExists TableName CreateSequenceOptionListOpt
+	{
+		$$ = &ast.CreateSequenceStmt{
+			IfNotExists: $3.(bool),
+			Name: $4.(*ast.TableName),
+			Options: $5.([]*ast.SequenceOption),
+		}
+	}
+
+CreateSequenceOptionListOpt:
+	{
+		$$ = []*ast.SequenceOption{}
+	}
+|       SequenceOptionList
+
+SequenceOptionList:
+	SequenceOption
+	{
+		$$ = []*ast.SequenceOption{$1.(*ast.SequenceOption)}
+	}
+|	SequenceOptionList SequenceOption
+	{
+		$$ = append($1.([]*ast.SequenceOption), $2.(*ast.SequenceOption))
+	}
+
+SequenceOption:
+	"INCREMENT" "BY" NUM
+	{
+		$$ = &ast.SequenceOption{ Tp:ast.SequenceOptionIncrementBy, IntValue: $3.(int64),}
+	}
+|	"START" "WITH" NUM
```

Comment:
Consider supporting these additional syntax:

* START = *num*
* START *num*

Again, the number can be negative.

Reply:
Addressed.